container/go/pkg/compat/ociToLegacy.go

```diff
@@ -24,12 +24,14 @@ import (
 	"strconv"
 
 	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/layout"
 	"github.com/pkg/errors"
 )
 
 // Extension for layers and config files that are made symlinks
 const targzExt = ".tar.gz"
 const configExt = "config.json"
```

Comment:
For config & manifest, these are not really extensions but the name of the files themselves. I would remove the "Ext" from the names of the variables. Suggestion `legacyConfigFile` & `legacyManifestFile` but feel free to choose something else.